### ISSUE TITLE
[Need Fix]Is there any possible to change set the txn data from bytes to args (i.e., BoxedABI)?

### ISSUE THREAD

**Issue by Kenun99** [fuzzland/ityfuzz#91]
I failed to update the  txn data (`ABI`) in the `EVMInput` using the method `set_bytes`. 
https://github.com/fuzzland/ityfuzz/blob/e5826df71d7293e02d8f83089fdb6ad363554071/src/evm/abi.rs#L220

It seems that the concolic engine also needs to update the abi data of the current input. 
https://github.com/fuzzland/ityfuzz/blob/e5826df71d7293e02d8f83089fdb6ad363554071/src/evm/concolic/concolic_host.rs#L1017

However, although I have activated the code after removing the annotations, `fn set_bytes()` still failed to change the input data. 
I was wondering whether it is possible to let me ask for any suggestions? 
Does the `set_bytes` in `ABI` work well?

The concrete example is shown here. The below code is inserted into the entry of `fn evaluate_input_events` in `ityfuzz/src/fuzzer.rs`. However, the two print commands output the sample.

```rust
tx_bytes = [0, 0, 0, 0] // example data
let mut cu_input = input.clone();
println!("got it: {:02x?}", cu_input.get_data_abi());
cu_input
    .get_data_abi_mut()
    .as_mut()
    .unwrap()
    .set_bytes(tx_bytes.to_vec());
println!("got it: {:02x?}", cu_input.get_data_abi());
```

**Comment by shouc** [comment]
It works for simple types (uint256, address, bytes, etc.) but does not work for tuples / arrays. We are still working on supporting them.  

This should work
```
        let mut abi = get_abi_type_boxed(&String::from("int256"));
        abi.set_bytes(vec![0xff; 32]);
        println!("abi: {:?}", abi);

```

**Comment by ByteSecurity** [comment]
fix #121
